models/fates/inst/create_1x1_ref_case.sh

```diff
@@ -18,15 +18,20 @@
 # site_lat_c =   9.1543, 5.07389,  -2.60909722,  1.4368,   4.1865,  15.6324
 # site_lon_c = 280.1539, 8.85472, 299.7907,     28.5826, 114.017,   99.217
 #=============================================================================================
-#export NETCDF_HOME=/usr/local/  # is this nescessary?
+export NETCDF_HOME=/usr/local/  
+export NETCDF_PATH=${NETCDF_HOME}
```

Comment:
@tonygardella @mdietze I think we need to be clearer with this build script that it is a template and is not generic enough to download and run everywhere.  For example ,I don't need to export my netCDF paths....but it was clear from the other day that for your current setup you do.  So this should be something listed as "optional"